Sources/Runner/MarathonScriptManager.swift

```diff
@@ -33,15 +33,13 @@ func getScriptManager(_ logger: Logger) throws -> ScriptManager {
     return try ScriptManager(folder: scriptFolder, packageManager: packageManager, printer: printer, config: config)
 }
 
-private func makeProgressPrintingFunction(logger: Logger) -> VerbosePrintFunction {
-    return { (messageExpression: () -> String) in
+private func makeProgressPrintingFunction(logger: Logger) -> VerbosePrintFunction { { (messageExpression: () -> String) in
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Runner/MarathonScriptManager.swift;
  Line: 36;
-->

- :warning: Line should be 120 characters or less: currently 122 characters (`line_length`)